lib/modules/commentSortBy.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+
+import _ from 'lodash';
+import { $ } from '../vendor';
+import { Module } from '../core/module';
+import { rawLocale } from '../utils/localization';
+import {
+	string,
+	watchForRedditEvents,
+} from '../utils';
+
+export const module: Module<*> = new Module('commentSortBy');
+
+module.beforeLoad = () => {
+	watchForRedditEvents('postModTools', () => {
+		applyToPost();
+	});
+};
+
+module.moduleName = 'commentSortBy';
+module.category = 'commentsCategory';
+module.disabledByDefault = false;
+module.description = 'addCommentSortPrefMenuItemDesc';
+
+export class SortByPrefLink {
+	static defaultLinkElement = (e => () => e().cloneNode(true))(_.once(() => SortByPrefLink.buildTagElement()));
+	static buildTagElement = () => string.html`
+			<span class="RESUserTag">
+				<a
```

Comment:
Do you mean `${string.safe(SettingsNavigation.makeUrlHashLink(module.moduleID, 'hardIgnore', ' configure ', 'gearIcon'))}`
It only links to the extension settings, not to the Reddit settings. Also my gear has an opacity of 0.3 otherwise it is too obnoxious over the white background.